### ISSUE TITLE
Better detection of editable RRULEs

### ISSUE THREAD

**Issue by samdmarshall** [pimutils/khal#662]
I have an event in my calendar that repeats on the first of the month every two months (e.g.: Feb 1st, April 1st, June 1st, etc). `khal` seems able to represent this on the calendar correctly but doesn't provide a way to see this rule in the event editor. I don't know if this is a bug in understanding the recurrence rule of the event or that there has been no UI built to represent this yet.

**Comment by geier** [comment]
khal > 0.9.4 (which you seem to have) should be able to create and display those recurrence rules.
Can you post the RRULE part of the event?

**Comment by samdmarshall** [comment]
```
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//Apple Inc.//Mac OS X 10.11.1//EN
CALSCALE:GREGORIAN
BEGIN:VEVENT
CREATED:20151205T003404Z
UID:5CBC17D2-4570-4DA6-B1C0-3EF75ED4F337
RRULE:FREQ=MONTHLY;INTERVAL=2;BYDAY=MO,TU,WE,TH,FR,SA,SU;BYSETPOS=1
DTEND;TZID=America/New_York:20160201T100000
SUMMARY:Renew Server SSL Certificate
DTSTART;TZID=America/New_York:20160201T090000
DTSTAMP:20151205T003701Z
SEQUENCE:0
TRANSP:OPAQUE
X-APPLE-TRAVEL-ADVISORY-BEHAVIOR:AUTOMATIC
BEGIN:VALARM
X-WR-ALARMUID:37843197-2021-46D5-9D20-3AB5B683BB30
UID:37843197-2021-46D5-9D20-3AB5B683BB30
TRIGGER;VALUE=DURATION:-P1D
DESCRIPTION:Event reminder
ACTION:DISPLAY
END:VALARM
END:VEVENT
END:VCALENDAR
```

**Comment by geier** [comment]
Thanks. The problem here is the included BYDAY and BYSETPOS, we currently mark them as not reproducable because they allow pretty bizarre recurrence rules. The variants used here are the defaults though and could be omitted. We should recognize these cases and show it as editable.

I'll change the title accordingly.

**Comment by geier** [comment]
should be fixed with  a1cabba

**Comment by samdmarshall** [comment]
awesome! thanks!!
